src/topBar/index.tsx

```diff
@@ -22,38 +23,51 @@ const TopBar = ({
   leftItem,
   rightItem,
   centerItem,
-}:TopBarProps) => (
-  <header
-    className={cc([prefix({
-      topBar: true,
-      'topBar--fixed': fixed,
-      'topBar--bgShadedTransparent': bgShadedTransparent,
-      'topBar--bgTransparent': bgTransparent,
-    }), className])}
-  >
-    {
-      leftItem ? (
-        <div className={cc(prefix({ 'topBar-left': true }))}>
-          {leftItem}
-        </div>
-      ) : null
-    }
-    {
-      centerItem ? (
-        <div className={cc(prefix({ 'topBar-center': true }))}>
-          {centerItem}
-        </div>
-      ) : null
-    }
-    {
-      rightItem ? (
-        <div className={cc(prefix({ 'topBar-right': true }))}>
-          {rightItem}
-        </div>
-      ) : null
-    }
-    <style jsx>{style}</style>
-  </header>
-)
+  innerWrapperClassName = null,
+}:TopBarProps) => {
+  const Wrapper = Boolean(innerWrapperClassName) ? 'div' : Fragment
+  const wrapperProps: { className?: string } = {}
+  if (Boolean(innerWrapperClassName)) {
+    wrapperProps.className = cc([innerWrapperClassName])
+  }
+  return (
+    <header
+      className={cc([
+        {
+          topBar: true,
```

Comment:
why not `'kirk-topBar': true,`?

Reply:
Because I forgot to add it back when I removed the `prefix` helper